runelite-client/src/main/java/net/runelite/client/plugins/config/ConfigPanel.java

```diff
@@ -160,8 +169,8 @@ public void changedUpdate(DocumentEvent e)
 			}
 		});
 
-		setBorder(new EmptyBorder(10, 10, 10, 10));
-		setLayout(new DynamicGridLayout(0, 1, 0, 5));
+		setBorder(new EmptyBorder(0, 0, 0, 0));
```

Comment:
Do you even need to set a border if it's size is 0?

Reply:
This just overrides the border set by the PluginPanel superclass.

Reply:
I think setBorder(null) does that aswell, not sure what the performance difference here is but, easier to read.

Reply:
Good point, that should've been obvious. I'll fix this.